src/AnnotatedGroundProgram.cpp

```diff
@@ -530,7 +530,7 @@ void AnnotatedGroundProgram::computeAdditionalDependencies()
                 for (int i = 0; i < ea.inputs.size(); ++i) {
                     if (ea.pluginAtom->getInputType(i) == PluginAtom::PREDICATE) {
                         // polarity check
-                        if (ctx->config.getOption("FLPDecisionCriterionEM") && (!b.isNaf() && prop.isAntimonotonic(i) || b.isNaf() && prop.isMonotonic(i))) {
+                        if (ctx->config.getOption("FLPDecisionCriterionEM") && ((!b.isNaf() && prop.isAntimonotonic(i)) || (b.isNaf() && prop.isMonotonic(i)))) {
```

Comment:
This is correct, but for readability we could also add an extra `const bool antimonotonic = <same condition>;` as above. The compiler will optimize it away anyways.